### PR TITLE
osgeo-email.md: language fixes (and a suggestion)

### PR DESCRIPTION
**Pull request by florisvdh**
Please check whether these changes are all justified, I'm no native English speaker :wink: 

After re-reading, a general suggestion - but I consider it to be rather a detail: GRASS GIS is used as (sole) OSGeo GIS example in two paragraphs before embarking on connections with several other OSGeo projects in a next paragraph. An alternative could be to already mention some other projects as well at an early stage in the text.

E.g. at '_integrates with established OSGeo projects such as [GRASS GIS](https://grasswiki.osgeo.org/wiki/R_statistics)_' you could enumerate a few more already.

### CODE REVIEW

**Comment by florisvdh** [comment]
@Robinlovelace ignore the general comment here; I've copied it directly in your PR at r-spatial/discuss

File: osgeo-email.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ Dear OSGeo Incubation Committee,
 
 We would like to apply, as the 'R-Spatial' community, to become an OSGeo affiliated organisation.
 
-We are a diverse group with a shared interest in developing free and open tools for the reproducible analysis of geographic data. R is a popular and rapidly growing language for statistical computing and 'data science'. It is already part of OSGeo ecosystem: R ships with the OSGeo Live distribution, integrates with established OSGeo projects such as [GRASS GIS](https://grasswiki.osgeo.org/wiki/R_statistics), and (now slightly dated) tutorials listed on OSGeo's [old website](http://old.www.osgeo.org/educational_content).
+We are a diverse group with a shared interest in developing free and open tools for the reproducible analysis of geographic data. R is a popular and rapidly growing language for statistical computing and 'data science'. It is already part of the OSGeo ecosystem: R ships with the OSGeo Live distribution, integrates with established OSGeo projects such as [GRASS GIS](https://grasswiki.osgeo.org/wiki/R_statistics), and (now slightly dated) tutorials listed on OSGeo's [old website](http://old.www.osgeo.org/educational_content).
```
Comment (Robinlovelace) [inline osgeo-email.md:9]: 👍

File: osgeo-email.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ Dear OSGeo Incubation Committee,
...
+Continued development and innovation in R-OSGeo links are illustrated the [qgisprocess](https://github.com/paleolimbot/qgisprocess) package, which motivated positive changes in the QGIS source code (see [github.com/paleolimbot/qgisprocess/issues/21](https://github.com/paleolimbot/qgisprocess/issues/21)). The R-Spatial community relies on the OSGeo projects GDAL, PROJ and GEOS for data access and geographic operations. Core R-Spatial packages `sf`, `raster` and `terra` use bindings to the libraries for much of the heavy lifting and many thousands of people using R for spatial research (often without knowing) run OSGeo support code every day. We would like to support the ongoing work of these vital components of the wider community that is represented by the OSGeo-affiliated conference series FOSS4G. We also anticipate benefits from being part of the wider OSGeo community and would like to be more active members of the wider movement advocating free and open source software for geospatial.
```
Comment (Robinlovelace) [inline osgeo-email.md:13]: 👍

**Review by Robinlovelace — APPROVED** [review]

**Comment by Robinlovelace** [comment]
Many thanks @florisvdh - do you want to add your name to the list of 'R-Spatial developers'?

**Comment by florisvdh** [comment]
@Robinlovelace obviously I don't consider myself as much an R-Spatial developer as other involved people, even though I make some small contributions here and there; probably my 'least small' contribution was adding `stringexecGRASS()` to `rgrass7` lately - BTW that's another package not yet officially part of github.com/r-spatial.

So, if you entitle the signing people as 'the R-spatial developers', then I'd rather not be in the list. I would really consider that as taking more responsability and invest more time in contributing / package management / r-spatial task view etc etc, which I can't at the moment (for the moment I rather aim at demonstrating stuff to R users, notably colleagues). But if you think broader support for the letter is needed and you would wide up to 'R-spatial contributors' then of course I'm willing to support with my name - but so would be many others, probably, in the R-spatial community, so others could be invited as well and then I think that makes a stronger case.

That's my view, but it's perfectly OK to leave things as is.

**Comment by Robinlovelace** [comment]
Thanks @florisvdh, lots to think about, we can always adapt plans and be flexible about definitions, seems this process is generating some important ideas/definitions!

**Pull request merged**